web/toc.md

```diff
@@ -0,0 +1,133 @@
+---
+ebook:
+  theme: github-light.css
+  title: K
+  author: Runtime Verification
+
+title: "K"
+author: Runtime Verification
+output:
+  pdf_document:
+    latex_engine: xelatex
+---
+
+# K
+
+# Table of Contents
+
+- [Homepage](/web/pages/index.md)
+- [K Tool Binaries](/web/pages/downloads.md)
+- [K Tutorial](/k-distribution/k-tutorial/README.md)
+  - [Section 1: Basic K Concepts](/k-distribution/k-tutorial/1_basic/README.md)
+    - [Lesson 1.1: Setting up a K Environment](/k-distribution/k-tutorial/1_basic/01_installing/README.md)
+    - [Lesson 1.2: Basics of Functional K](/k-distribution/k-tutorial/1_basic/02_basics/README.md)
+    - [Lesson 1.3: BNF Syntax and Parser Generation](/k-distribution/k-tutorial/1_basic/03_parsing/README.md)
+    - [Lesson 1.4: Disambiguating Parses](/k-distribution/k-tutorial/1_basic/04_disambiguation/README.md)
+    - [Lesson 1.5: Modules, Imports, and Requires](/k-distribution/k-tutorial/1_basic/05_modules/README.md)
+    - [Lesson 1.6: Integers and Booleans](/k-distribution/k-tutorial/1_basic/06_ints_and_bools/README.md)
+    - [Lesson 1.7: Side Conditions and Rule Priority](/k-distribution/k-tutorial/1_basic/07_side_conditions/README.md)
+    - [Lesson 1.8: Literate Programming with Markdown](/k-distribution/k-tutorial/1_basic/08_literate_programming/README.md)
+    - [Lesson 1.9: Unparsing and the format and color attributes](/k-distribution/k-tutorial/1_basic/09_unparsing/README.md)
+    - [Lesson 1.10: Strings](/k-distribution/k-tutorial/1_basic/10_strings/README.md)
+    - [Lesson 1.11: Casting Terms](/k-distribution/k-tutorial/1_basic/11_casts/README.md)
+    - [Lesson 1.12: Syntactic Lists](/k-distribution/k-tutorial/1_basic/12_syntactic_lists/README.md)
+    - [Lesson 1.13: Basics of K Rewriting](/k-distribution/k-tutorial/1_basic/13_rewrite_rules/README.md)
+    - [Lesson 1.14: Defining Evaluation Order](/k-distribution/k-tutorial/1_basic/14_evaluation_order/README.md)
+    - [Lesson 1.15: Configuration Declarations and Cell Nesting](/k-distribution/k-tutorial/1_basic/15_configurations/README.md)
+    - [Lesson 1.16: Maps, Semantic Lists, and Sets](/k-distribution/k-tutorial/1_basic/16_collections/README.md)
+    - [Lesson 1.17: Cell Multiplicity and Cell Collections](/k-distribution/k-tutorial/1_basic/17_cell_multiplicity/README.md)
+    - [Lesson 1.18: Term Equality and the Ternary Operator](/k-distribution/k-tutorial/1_basic/18_equality_and_conditionals/README.md)
+    - [Lesson 1.19: Debugging with GDB](/k-distribution/k-tutorial/1_basic/19_debugging/README.md)
+    - [Lesson 1.20: K Backends and the Haskell Backend](/k-distribution/k-tutorial/1_basic/20_backends/README.md)
+    - [Lesson 1.21: Unification and Symbolic Execution](/k-distribution/k-tutorial/1_basic/21_symbolic_execution/README.md)
```

Comment:
We should probably only include the lessons that are actually done here, to avoid confusion.

Reply:
Hi @dwightguth . I have removed the unimplemented lessons. Please help review the PR again when you have time. Thanks!